modules/glb/README.md

```diff
@@ -1,29 +1,25 @@
-# Module CIS GLB
+# CIS Global Load Balancer (GLB) Module
 
-This module is used to provision Global Load Balancers, Origin pools and monitors.
+This module is used to provision Global Load Balancer that include load balancers, origin pools and health checks.
```

Comment:
lowercase global load balancer here as everywhere else

Reply:
made the required changes